src/OpenTelemetry.Exporter.Prometheus.AspNetCore/PrometheusAspNetCoreOptions.cs

```diff
@@ -17,6 +17,15 @@ public class PrometheusAspNetCoreOptions
     /// </summary>
     public string ScrapeEndpointPath { get; set; } = DefaultScrapeEndpointPath;
 
+    /// <summary>
+    /// Gets or sets a value indicating whether addition of _total suffix for counter metric names is disabled. Default value: <see langword="false"/>.
+    /// </summary>
+    public bool DisableTotalNameSuffixForCounters
```

Comment:
Add this for [PrometheusHttpListenerOptions](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.Prometheus.HttpListener/PrometheusHttpListenerOptions.cs) as well.

Reply:
Add a CHANGELOG entry as well.

Reply:
done